gtsfm/loader/astronet_loader.py

```diff
@@ -118,13 +122,19 @@ def __init__(
         self._num_imgs = len(self._image_paths)
         logger.info("AstroNet loader found and loaded %d images and %d tracks.", self._num_imgs, self.num_sfmtracks)
 
+    @property
+    def gt_gtsfm_data(self):
```

Comment:
nit on name: just gt_data or get_gt_data might suffice

Reply:
Usually with properties, you name it after the attribute you're returning.

Reply:
I dont feel strongly, so you can keep it this way, though I tend to name functions based on what they are returning semantically, not the datatype.